docs/api-docs/catalog/price-list-overview.md

```diff
@@ -4,9 +4,7 @@
 	<ul>

```

Comment:
This table of contents needs to be converted to markdown. Please refer to the setup in other pages for the correct formatting.